config/example_wod_profiles.json

```diff
@@ -29,11 +29,11 @@
                         "Phosphate_row_size": "Phosphate_row_size",
                         "tCO2": "DIC",
                         "tCO2_row_size": "DIC_row_size",
-                        "Salinity_WODflag" : "Salinity_flag",
-                        "Temperature_WODflag" : "Temperature_flag",
-                        "Phosphate_WODflag" : "Phosphate_flag",
-                        "tCO2_WODflag" : "DIC_flag",
-                        "Oxygen_WODflag" : "Oxygen_flag"
+                        "Salinity_wodflag" : "Salinity_flag",
+                        "Temperature_wodflag" : "Temperature_flag",
+                        "Phosphate_wodflag" : "Phosphate_flag",
+                        "tCO2_wodflag" : "DIC_flag",
+                        "Oxygen_wodflag" : "Oxygen_flag"
```

Comment:
I'm confused by this change I made. I though the LHS was the variable name in the file and the RHS is the target COAsT name. How can this stuff work if I've not edited the netcdf file contents?